components/Settings.js

```diff
@@ -44,6 +44,16 @@ class Settings extends React.Component {
             enabled={this.props.enabled.dropShadow}
             onChange={this.props.onChange.bind(null, 'dropShadow')}
           />
+          <Slider
+            label="Drop shadow offset-y"
```

Comment:
I think we should change these to 'Drop shadow (offset-y)' to be consistent with Padding (verticle), etc.